Tests are failing on Ubuntu 18.04 with IDA 7.0

Hello. I wanted to try this project, after some struggling with compilation and dependancies i successfully built it. By the step 5 (in Linux section) i headed to [Integration Tests](https://github.com/trailofbits/mcsema/blob/master/tests/MakingTests.md) and attempted to execute them as described, however i got this:
```
QXcbConnection: Could not connect to display 
Aborted (core dumped)
Traceback (most recent call last):
  File "/media/UserData/Youda/CVUT/DIP/Implementace/frameworks/mcsema-venv/local/lib/python2.7/site-packages/mcsema_disass-2.0-py2.7.egg/mcsema_disass/ida/disass.py", line 77, in execute
    cwd=os.path.dirname(__file__))
  File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
    raise CalledProcessError(retcode, cmd)
CalledProcessError: Command '/home/youda/idafree-7.0/idal64 -B -S"/media/UserData/Youda/CVUT/DIP/Implementace/frameworks/mcsema-venv/local/lib/python2.7/site-packages/mcsema_disass-2.0-py2.7.egg/mcsema_disass/ida/get_cfg.py --output /media/UserData/Youda/CVUT/DIP/Implementace/frameworks/mcsema-venv/remill/tools/mcsema/tests/test_suite_generator/build/src/linux/arc4/arc4_amd64.cfg --log_file /dev/null --arch amd64 --os linux --entrypoint main" /tmp/tmp3KaYK3/arc4' returned non-zero exit status 134
Could not generate a CFG. Try using the --log_file option to see an error log.
Traceback (most recent call last):
  File "/media/UserData/Youda/CVUT/DIP/Implementace/frameworks/mcsema-venv/bin/mcsema-disass", line 11, in <module>
    load_entry_point('mcsema-disass==2.0', 'console_scripts', 'mcsema-disass')()
  File "/media/UserData/Youda/CVUT/DIP/Implementace/frameworks/mcsema-venv/local/lib/python2.7/site-packages/mcsema_disass-2.0-py2.7.egg/mcsema_disass/__main__.py", line 169, in main
    if 0 == os.path.getsize(args.output):
  File "/media/UserData/Youda/CVUT/DIP/Implementace/frameworks/mcsema-venv/lib/python2.7/genericpath.py", line 57, in getsize
    return os.stat(filename).st_size
OSError: [Errno 2] No such file or directory: '/media/UserData/Youda/CVUT/DIP/Implementace/frameworks/mcsema-venv/remill/tools/mcsema/tests/test_suite_generator/build/src/linux/arc4/arc4_amd64.cfg'
src/linux/arc4/CMakeFiles/arc4_amd64_bc.dir/build.make:64: recipe for target 'src/linux/arc4/arc4_amd64.cfg' failed
make[2]: *** [src/linux/arc4/arc4_amd64.cfg] Error 1
CMakeFiles/Makefile2:100: recipe for target 'src/linux/arc4/CMakeFiles/arc4_amd64_bc.dir/all' failed
make[1]: *** [src/linux/arc4/CMakeFiles/arc4_amd64_bc.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
```

I see that your examples are all for IDA 6.9. Could it be some incompatibility with the newer version? Also your scripts search for executable `idal64`, but in IDA 7 it's called `ida64`.
IDA 6.9 is no longer possible to obtain, can you please update your project to support IDA 7 ?

--- a/tests/MakingTests.md
+++ b/tests/MakingTests.md
@@ -12,15 +12,24 @@ cd test_suite_generator
 mkdir build
 cd build
 export TRAILOFBITS_LIBRARIES=<path to remill libraries>
-export IDAL64_PATH=<path to idal64>
-export CMAKE_PROGRAM_PATH=$(dirname ${IDAL64_PATH})
+export IDAT64_PATH=<path to idat64>
+export CMAKE_PROGRAM_PATH=$(dirname ${IDAT64_PATH})
 cmake ..
 make
 make install #install to ../test_suite
 
 #example: 
 #export TRAILOFBITS_LIBRARIES=/store/artem/git/remill/remill-build/libraries/
-#export IDAL64_PATH=/home/artem/ida-6.9/idal64
-#export CMAKE_PROGRAM_PATH=$(dirname ${IDAL64_PATH})
+#export IDAT64_PATH=/home/artem/ida-7.1/idat64
+#export CMAKE_PROGRAM_PATH=$(dirname ${IDAT64_PATH})
 #cmake ..
+```
+
+# Running the tests
+
+After building and installing, you can do:
+
+```sh
+cd test_suite_generator/test_suite
+python2 start.py
 ```

--- a/tests/test_suite_generator/CMakeLists.txt
+++ b/tests/test_suite_generator/CMakeLists.txt
@@ -140,7 +140,7 @@ function (ExportCFG target_name architecture binary_path)
 
   set(output_file "${target_name}_${architecture}.cfg")
 
-  if ("${IDAL64_PATH}" STREQUAL "IDAL64_PATH-NOTFOUND")
+  if ("${IDAT64_PATH}" STREQUAL "IDAT64_PATH-NOTFOUND")
     #Read CFG from pre-built CFG repository
     #configure_file("${MCSEMA_PREBUILT_CFG_PATH}/${architecture}/${platform}/cfg/${target_name}"
     #               "${output_file}"
@@ -155,7 +155,7 @@ function (ExportCFG target_name architecture binary_path)
   else ()
     add_custom_command(
       OUTPUT "${output_file}"
-      COMMAND "${MCSEMADISASS_PATH}" --disassembler "${IDAL64_PATH}" --arch "${architecture}" --os "${platform}" --binary "${binary_path}" --output "${output_file}" --entrypoint main
+      COMMAND "${MCSEMADISASS_PATH}" --disassembler "${IDAT64_PATH}" --arch "${architecture}" --os "${platform}" --binary "${binary_path}" --output "${output_file}" --entrypoint main
       DEPENDS "${binary_path}"
       VERBATIM
     )
@@ -354,11 +354,11 @@ if ("${MCSEMALIFT_PATH}" STREQUAL "MCSEMALIFT_PATH-NOTFOUND")
   message(FATAL_ERROR "Failed to locate the mcsema-lift${LLVM_VERSION_SUFFIX} executable!")
 endif ()
 
-find_program(IDAL64_PATH "idal64")
-if ("${IDAL64_PATH}" STREQUAL "IDAL64_PATH-NOTFOUND")
-  message(WARNING "Failed to locate the idal64 executable! Will not test CFG recovery.")
+find_program(IDAT64_PATH "idat64")
+if ("${IDAT64_PATH}" STREQUAL "IDAT64_PATH-NOTFOUND")
+  message(WARNING "Failed to locate the idat64 executable! Will not test CFG recovery.")
   if(NOT DEFINED MCSEMA_PREBUILT_CFG_PATH)
-    message(FATAL_ERROR "IDAL64_PATH is not defined and MCSEMA_PREBUILT_CFG_PATH is not defined (use -DMCSEMA_PREBUILT_CFG_PATH=<path>)")
+    message(FATAL_ERROR "IDAT64_PATH is not defined and MCSEMA_PREBUILT_CFG_PATH is not defined (use -DMCSEMA_PREBUILT_CFG_PATH=<path>)")
   endif()
   get_filename_component(MCSEMA_PREBUILT_CFG_PATH "${MCSEMA_PREBUILT_CFG_PATH}" ABSOLUTE)
   if(NOT EXISTS "${MCSEMA_PREBUILT_CFG_PATH}")
@@ -374,8 +374,8 @@ endif ()
 message("Toolset")
 message(" > mcsema-lift: ${MCSEMALIFT_PATH}")
 message(" > clang: ${CLANG_PATH}")
-if (NOT "${IDAL64_PATH}" STREQUAL "IDAL64_PATH-NOTFOUND")
-  message(" > idal64: ${IDAL64_PATH}")
+if (NOT "${IDAT64_PATH}" STREQUAL "IDAT64_PATH-NOTFOUND")
+  message(" > idat64: ${IDAT64_PATH}")
 else()
   message(" > using prebuilt cfgs: ${MCSEMA_PREBUILT_CFG_PATH}")
 endif ()

